src/java.compiler/share/classes/javax/lang/model/util/ElementKindVisitorPreview.java

```diff
@@ -86,4 +86,21 @@ protected ElementKindVisitorPreview() {
     protected ElementKindVisitorPreview(R defaultValue) {
         super(defaultValue);
     }
+
+    /**
+     * {@inheritDoc ElementKindVisitor6}
```

Comment:
If possible, would be good to add some minimal testing/use of the element kind visitors on component local variables.

Reply:
Tests added: https://github.com/openjdk/jdk/pull/18509/commits/e09306885e5c6cd2aba7779025fc7efcb8991e8e

Thanks!